HLeithner/joomla-cms
# Prepared plg content contact

### Summary of Changes
Updated SQL queries to prepared statements and made some cleanups around the queries.


### Testing Instructions
Use the plugin in all ways you can think of.

* Create users
* Create contacts linked to the users
* Create articles linked to different users
* Contact should be shown at the article
* use different settings in the plugin



### Expected result
Nothing changed.
